rust/frontend/src/api.rs

```diff
@@ -0,0 +1,55 @@
+use chroma_config::Configurable;
+use chroma_error::ChromaError;
+use chroma_sysdb::sysdb;
+use chroma_types::CreateDatabaseResponse;
+
+use crate::config::FrontEndConfig;
+
+#[async_trait::async_trait]
```

Comment:
There is only one concrete impl of this ever, I don't see a reason to have a trait.